Observation-ISH.StructureDefinition.json

```diff
@@ -45,6 +45,18 @@
   "derivation": "constraint",
   "differential": {
     "element": [
+      {
+        "id": "Observation",
+        "path": "Observation",
+        "constraint": [
+          {
+            "key": "amplifikation",
+            "severity": "error",
+            "human": "If Observation concerns MET, ALK or ROS1 then a different VS shall be bound",
+            "expression": "Observation.code.coding.code.where(code = ‘MET’) implies Observation.component.valueCodableConcept.coding.system.where(system = 'http://uk-koeln.de/fhir/ValueSet/nNGM/ergebnis-cish-met').exists() and Observation.code.coding.code.where(code = ‘MET’) implies Observation.component.valueCodableConcept.coding.system.where(system = ‘http://uk-koeln.de/fhir/ValueSet/amplification-met’).exists()"

```

Comment:
Selecting polymorphic elements in FHIRPath is done differently, see http://hl7.org/fhirpath/N1/#paths-and-polymorphic-items

valueCodeableConcept is not allowed. ofType should be used.

Reply:
changed it to ofType